dev/io.openliberty.xmlws.3.0.internal.tools/src/com/ibm/ws/jaxws/tools/WsGen.java

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
```

Comment:
I would like to explore the possibility of using Constants in JaxWsToolsConstants both the `com.ibm.ws.jaxws.tools` and this bundle to allow us not to duplicate this code, and override the values in `JaxWsToolsConstants` in this bundle. Since the only differences are updating the properties and adding a new SAAJ property, this should be possible, but I ran into some difficulties in my first attempt and so for the sake of time I duplicated only the code that needed changes.